src/StatusQ/Controls/StatusFloatingButtonsSelector.qml

```diff
@@ -0,0 +1,121 @@
+   \since StatusQ.Components 0.1
+   \brief The StatusModalFloatingButtonsSelector provides a template for creating a selectable buttons list
+   this list can be parially hidden and the rest of the items are shown under the more button in a popup
+
+   Example of how to use it:
+
+   \qml
+        StatusModalFloatingButtonsSelector {
+            id: floatingHeader
+            itemsModel: dummyAccountsModel
+            delegate: StatusAccountSelectorTag {
+                title: "name"
+                icon.name: "iconName"
+                isSelected: floatingHeader.currentIndex === index
+                visible: visibleIndices.includes(index)
+                onClicked: floatingHeader.currentIndex = index
+            }
+            popupMenuDelegate: StatusListItem {
+                implicitWidth: 272
+                title: "name"
+                onClicked: floatingHeader.itemSelected(index)
+                visible: !visibleIndices.includes(index)
+            }
+        }
+   \endqml
+
+   For a list of components available see StatusQ.
+*/
+Row {
+    id: floatingButtons
+
+    /*!
+       \qmlproperty delegate
+        This property represents the delgate of selectable items shown to the user.
+        Can be used to assign delegate to the buttons selector
+        \endqml
+    */
+    property alias delegate: itemSelectionRepeater.delegate
+    /*!
+       \qmlproperty popupMenuDelegate
+        This property represents the delgate of popupmenu fropm which items can be selected by the user.
+        Can be used to assign delegate to the popupmenu
+        \endqml
+    */
+    property alias popupMenuDelegate: popupMenuSelectionRepeater.delegate
+
+    /*!
+       \qmlproperty itemsModel
+        This property represents the model of selectable items shown to the user.
+        Can be used to assign selectable items in the buttons selector
+        \endqml
+    */
+    property var itemsModel
+    /*!
+       \qmlproperty visibleIndices
+        This property represents the indices from the selectable items that will visible to the user
+        Can be used to set visible items in the buttons selector
+        \endqml
+    */
+    property var visibleIndices: [0,1,2]
+    /*!
+       \qmlproperty currentIndex
+        This property represents the index of the currently selected item
+        Can be used to set the currnetly selected item in the buttons selector
+        \endqml
+    */
+    property int currentIndex: 0
+
+    function itemSelected(index) {
+        visibleIndices = [0,1,index]
+        floatingButtons.currentIndex = index
+        popupMenu.close()
+    }
+
+    height: 32
+    spacing: 12
+    clip: true
+
+    Repeater {
+        id: itemSelectionRepeater
+        model: itemsModel
+    }
+
+    StatusClickableTag {
+        visible: itemsModel.count > 3
+        icon.name: "more"
+        icon.background.color: "transparent"
+        onClicked: popupMenu.popup(x, y + height + 8)
+    }
```

Comment:
The usage of this thing makes it semantically even more a button than a tag :D (sorry)